x/evm/client/cli/query.go

```diff
@@ -17,11 +17,11 @@ package cli
 
```

Comment:
Use autocli, instead of cli/query define

Reply:
I'll check this later

Reply:
https://github.com/xpladev/ethermint/issues/15